src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -179,12 +179,64 @@
     gap: 16px;
   }
 
+  .overview {
+    margin-top: 6px;
+  }
+
+  .second-btn {
+    margin-left: 12px;
+  }
+
   :global(.phone),
   :global(.phone-xs) {
     .btn-1 {
       width: 100%;
       margin-top: 8px;
     }
+
+    .card {
+      --margin: 6px;
+    }
+
+    .setting-mobile {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+    }
+
+    .txt-right {
+      margin-top: 12px;
+      display: flex;
+      align-items: center;
+    }
+
+    .btn-2 {
+      --v-padding: 7px;
+    }
+
+    .nowrap {
+      margin-top: 12px;
+    }
+
+    .cancel {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
```

Comment:
This issue was not resolved https://github.com/santiment/san-webkit/pull/93#discussion_r1083955383